ext/iperf3/iperf_api.c

```diff
@@ -60,8 +60,8 @@
 #include <sys/time.h>
 
 #if defined(CONFIG_NRF_IPERF3_INTEGRATION)
-#if defined (CONFIG_NRF_MODEM_LIB_TRACE_ENABLED) && defined(CONFIG_AT_CMD)
-#include <modem/at_cmd.h>
+#if defined (CONFIG_NRF_MODEM_LIB_TRACE_ENABLED)
+#include <nrf_modem_at.h>
 #endif
```

Comment:
Looks like this include is not even needed as there is one 30 lines below.

Reply:
Thank you, I removed the extra include